Jenkinsfile.nightly

```diff
@@ -43,11 +43,12 @@ try {
         pkgVersion = buildPkgVersion
         srcVersion = buildSrcVersion
         testSchema = [
-            ['test_ledger.py'],
-            ['test_vc.py'],
+            ['test_ledger.py', 'test_state_proof'],
```

Comment:
BTW maybe it makes sense to run `test_state_proof` on every build, not on nightly only? This looks like quite important functionality (maybe the most important for the end users).

Reply:
Done.